diff -Nur kalamaris-0.6.0.orig/kalamaris/kalamaris.cpp kalamaris-0.6.0/kalamaris/kalamaris.cpp
--- kalamaris-0.6.0.orig/kalamaris/kalamaris.cpp	Mon Jun  5 03:01:48 2000
+++ kalamaris-0.6.0/kalamaris/kalamaris.cpp	Wed Nov  1 20:28:07 2000
@@ -173,7 +173,7 @@
 
   QString tmpFile=tmpnam(0);
   saveFile(tmpFile);
-  if( KIO::NetAccess::upload( tmpFile, url ) ) unlink(tmpFile);
+  if( KIO::NetAccess::upload( tmpFile, url ) ) delete tmpFile;
 }
 
 void MainWindow::processCmdline()
